docs/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,212 @@
+---
+id: soap
+title: SOAP Connector
+sidebar_label: SOAP Connector
+description: The SOAP Connector allows you to connect your BPMN process with SOAP services.
+---
+
+:::note
+The **SOAP Connector** is only supported by Self-Managed Camunda 8 instances.
+:::
+
+**Simple Object Access Protocol (SOAP)** is a messaging protocol specification for exchanging structured
+information in the implementation of web services in computer networks.
+
+The **SOAP Connector** allows you to interact with [SOAP](https://en.wikipedia.org/wiki/SOAP) service endpoints
+from your BPMN process.
+
+## Prerequisites
+
+To use the **SOAP Connector**, ensure you have an active SOAP service.
+
+## Create a SOAP Connector task
+
+To use the **SOAP Connector** in your process, either change the type of an existing task by clicking on it and using
+the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu.
+Follow our [guide to using Connectors](/components/connectors/use-connectors/index.md) to learn more.
+
+## Connection
+
+Enter your SOAP service URL in the field **Service URL**, for example `https://myservice.com/service/MyService.wso`.
+
+## Authentication
+
+Select the authentication type from the **Authentication** dropdown.
+
+### None
+
+Use **None** if the SOAP service does not require authentication.
+
+### WSS username token
+
+Use **WSS username token** in the **Authentication** dropdown when the requested SOAP endpoint requires
+[username token extension](https://docs.oasis-open.org/wss/v1.1/wss-v1.1-spec-pr-UsernameTokenProfile-01.htm#_Toc104276211).
+
+Enter **Username**, **Password**, and indicate if the password is encoded.
+
+:::note
+The **SOAP Connector** currently supports only `SHA-1` password encoding.
+:::
+
+### WSS signature
+
+Use the **WSS signature** in the **Authentication** dropdown when the requested SOAP endpoint requires a message to be
+cryptographically signed with a [signature](http://docs.oasis-open.org/wss-m/wss/v1.1.1/cs01/wss-SOAPMessageSecurity-v1.1.1-cs01.html#_Toc307407954).
+
+Enter all necessary fields according to your service specification.
```

Comment:
We should be prepared to extend here based on customer feedback :P

Reply:
I don't have a valid example. Do you mind adding one?

Reply:
@jonathanlukas, do you have something already? Or can we resolve this?

Reply:
not yet